config/v1/types.go

```diff
@@ -5,19 +5,25 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-// ConfigMapReference references a configmap in the openshift-config namespace.
-type ConfigMapReference struct {
+// ConfigMapFileReference references a configmap in the openshift-config namespace.
+type ConfigMapFileReference struct {
 	Name string `json:"name"`
 	// Key allows pointing to a specific key/value inside of the configmap.  This is useful for logical file references.
 	Key string `json:"key,omitempty"`
 }
 
-// LocalSecretReference references a secret within the local namespace
-type LocalSecretReference struct {
-	// Name of the secret in the local namespace
+// ConfigMapNameReference references a config map in a specific namespace.
+// If a namespace is not specified at the point of use, the openshift-config namespace is assumed.
```

Comment:
too strict on namespace.  your own usage in this pull will be in openshift-config-managed.  I'm ok with indicating both possibilities with guidance about spec versus status and matching

Reply:
> too strict on namespace. your own usage in this pull will be in openshift-config-managed. I'm ok with indicating both possibilities with guidance about spec versus status and matching

Not sure what you mean here.  All the user supplied config (from IDPs) will come from `openshift-config`.  All the operator written config (primarily oauth metadata) will go to `openshift-config-managed`.  If you want it to say "if the usage is in spec, the default namespace is openshift-config unless otherwise stated, and if the usages is in status, the default namespace is openshift-config-managed unless otherwise stated."  But I am not sure if that is what you are asking for.